File: app/src/main/java/me/anon/model/Garden.java
package me.anon.model;

import android.annotation.SuppressLint;
import android.os.Parcel;
import android.os.Parcelable;

import java.util.ArrayList;

import kotlinx.android.parcel.Parcelize;

@SuppressLint("ParcelCreator")
@Parcelize
public class Garden implements Parcelable
{
	protected String name;
	protected ArrayList<String> plantIds;

	public String getName()
	{
		return name;
	}

	public void setName(String name)
	{
		this.name = name;
	}

	public ArrayList<String> getPlantIds()
	{
		return plantIds;
	}

	public void setPlantIds(ArrayList<String> plantIds)
	{
		this.plantIds = plantIds;
	}

	@Override public int describeContents()
	{
		return 0;
	}

	@Override public void writeToParcel(Parcel parcel, int i)
	{

	}
}

File: app/src/main/java/me/anon/model/Garden.kt
package me.anon.model

import android.os.Parcelable
import com.squareup.moshi.JsonClass
import kotlinx.android.parcel.Parcelize

/**
 * // TODO: Add class description
 */
@Parcelize
@JsonClass(generateAdapter = true)
class Garden(
	var name: String = "",
	var plantIds: ArrayList<String> = arrayListOf()
) : Parcelable
